pkg/config/config_template.yaml

```diff
@@ -212,6 +212,16 @@ api_key:
 #
 # python_version: 2
 
+## @param flare_stripped_keys - list of strings - optional
+## By default, the Agent removes known sensitive keys from Agent and Integrations yaml configs before
+## including them in the flare.
+## Use this parameter to define additional sensitive keys that the Agent should scrub from
+## the yaml files included in the flare.
+#
+# flare_stripped_keys:
+#   - "sensitive_key_1"
+#   - "sensitive_key_2"
```

Comment:
Shouldn't this be in `Common` section instead of `BothPythonPresent` ?

Reply:
you're 100% right, let me fix this

Reply:
Fixed